### PR TITLE
Implemented the playlist presentation item

### PR DESCRIPTION
**Pull request by rodrigopavezi**
[stage-3]

Moved the presentation modal and services to the common-app-components as per https://github.com/Rise-Vision/common-app-components/pull/42

@alex-deaconu @ezequielc please review.
cheers

### CODE REVIEW

File: js/controllers/ctr-presentation-list.js [hunk 1]
```diff
@@ -1,71 +0,0 @@
...
-    'presentation', '$loading', 'BaseList', '$filter',
-    function ($scope, $rootScope, presentation, $loading, BaseList, $filter) {
-      var DB_MAX_COUNT = 40; //number of records to load at a time
-
-      $scope.presentations = new BaseList(DB_MAX_COUNT);
-
-      $scope.search = {
-        sortBy: 'changeDate',
-        count: DB_MAX_COUNT,
-        reverse: true
```
Comment (alex-deaconu) [inline js/controllers/ctr-presentation-list.js]: @rodrigopavezi I don't think it's a good idea to remove this controller. If you notice, the sortBy and reverse parameters are different from what the Presentation Selector was using. 

As discussed in our Scrum meeting, I think we need to wait until we have a common service for lists and we can just refactor all at the same time. Thoughts? Thanks.

File: partials/playlist-item-modal.html [hunk 2]
```diff
@@ -17,6 +17,10 @@ <h3 class="modal-title">Edit Playlist Item</h3>
         {{widgetName}} <i class="fa fa-cog icon-right"></i>
       </a>
     </p>
+    <p ng-show="presentationType"><strong>Presentation:</strong>
+      <a id="presentationName" class="link add-left" href="" ng-click="playlistItemFactory.editPresentation(item)"  presentation-name="item.objectData">
```
Comment (alex-deaconu) [inline partials/playlist-item-modal.html]: @rodrigopavezi in this particular case we don't want to show the Presentation Name, but we want to have a TextBox where the user can insert any Presentation ID they wish (ie. Presentations from other companies). Thanks.

File: js/controllers/ctr-presentation-list.js (hunk 1 above)
Comment (rodrigopavezi) [inline js/controllers/ctr-presentation-list.js]: @alex-deaconu makes sense. I will moved it back and use another one on the Presentation Selector component. cheers

File: partials/playlist-item-modal.html (hunk 2 above)
Comment (rodrigopavezi) [inline partials/playlist-item-modal.html]: @alex-deaconu exactly like RVA? Do we need to validate the inserted presentation id? cheers

Comment (alex-deaconu) [inline partials/playlist-item-modal.html]: @rodrigopavezi I think we're good on validating the ID for now. Maybe requiredField validation would suffice? Thanks.

**Comment by rodrigopavezi** [comment]
@alex-deaconu I have made the changes, could you please review? Cheers

File: partials/playlist-item-modal.html [hunk 3]
```diff
@@ -1,12 +1,12 @@
-<div id="playlistItemModal">
+<div id="playlistItemModal" xmlns="http://www.w3.org/1999/html">
```
Comment (alex-deaconu) [inline partials/playlist-item-modal.html]: @rodrigopavezi curious what this is for? Thanks.

Comment (rodrigopavezi) [inline partials/playlist-item-modal.html]: @alex-deaconu me too. I think it was the something from IntelliJ that added it. I will remove it. cheers

File: partials/placeholder-playlist.html [hunk 4]
```diff
@@ -1,4 +1,19 @@
-<button id="addContentButton" class="btn btn-primary btn-block" ng-click="playlistItemFactory.add()">Add Playlist Content <i class="fa fa-plus icon-right"></i></button>
+<div class="" dropdown>
```
Comment (alex-deaconu) [inline partials/placeholder-playlist.html]: @rodrigopavezi I believe this class should be `btn-group` similar to https://github.com/Rise-Vision/editor-app/blob/master/partials/presentation-list.html#L10. Thanks.

**Comment by alex-deaconu** [comment]
@rodrigopavezi looks good :+1: . Thanks.

File: partials/placeholder-playlist.html (hunk 4 above)
Comment (rodrigopavezi) [inline partials/placeholder-playlist.html]: @alex-deaconu btn-group will make it look like this

screen shot 2015-09-29 at 4 41 18 pm

@ghislaineguerin should I add another class or leave it empty? cheers

Comment (alex-deaconu) [inline partials/placeholder-playlist.html]: @rodrigopavezi oh crap, never mind then I thought it was the class was just blank by mistake. Thanks.

Comment (rodrigopavezi) [inline partials/placeholder-playlist.html]: @alex-deaconu I will leave it like that and come back to it if @ghislaineguerin has any other option. cheers

**Comment by ghislaineguerin** [comment]
@rodrigopavezi why is there an empty class? can we remove that? otherwise it looks ok :+1:

**Comment by rodrigopavezi** [comment]
@ghislaineguerin sure, I will remove it on another PR. cheers

**Pull request merged**